README.md

```diff
@@ -313,3 +313,17 @@ Example usage:
 ```
 defraUtils.triggerDeploy((jenkinsDeployUrl, deployJobName, jenkinsToken, ['chartVersion':'1.0.0'])
 ```
+
+### notifySlackBuildFailure
+
+Sends a message to the ffc-notifications Slack workspace when the jenkins build fails.
+
+Takes 2 parameters:
```

Comment:
Tiny nitpick, the other descriptions in the file use the word rather than a number, i.e. this should be 'two' rather than '2'

Reply:
no problem. pushed an update just now